config/jobs/kube-oidc-proxy/kube-oidc-proxy-presubmits.yaml

```diff
@@ -42,9 +42,11 @@ presubmits:
     spec:
       containers:
       - image: golang:1.12.1
-        workingDir: /go/src/github.com/jetstack/kube-oidc-proxy/demo
+        workingDir: /go/src/github.com/jetstack/kube-oidc-proxy
```

Comment:
I don't think we should be specifying workingDir at all - afaik, it does *not* influence where your code is checked out to, and right now I think Prow is actually ignoring this option altogether.

Reply:
Sure, will remove all occurrences from the file if that's the case